commitlint/action.yaml

```diff
@@ -1,109 +1,21 @@
 name: Commitlint
 description: GitHub Action for running commitlint within the pre-commit action.
 inputs:
-  config-file:
-    required: true
-    description: The config file to present to commitlint-github-action
-    default: .commitlintrc.yaml
   turo-conventional-commit:
     required: true
-    description: Set this to "false" to customize conventional commit config
-    default: "true"
-  restore-config:
-    description: Set to "false" to skip the configuration restore step
+    description: Install @open-turo/commitlint-config-conventional
     default: "true"
 
 runs:
   using: composite
   steps:
-    - name: Install @commitlint/cli
-      shell: bash
-      run: |
-        npm install --no-save --no-package-lock --no-audit --no-fund \
-          --prefix "$RUNNER_TEMP" "@commitlint/cli@^18"
-        echo "$RUNNER_TEMP/node_modules/.bin" >> $GITHUB_PATH
-        echo "NODE_PATH=$RUNNER_TEMP/node_modules" >> $GITHUB_ENV
-    - name: Use default conventional commit config
+    - name: Install extra plugins
       if: inputs.turo-conventional-commit
       shell: bash
       run: |
-        # Write default commitlint config in the workspace.
-        echo "extends: [\"@open-turo/commitlint-config-conventional\"]" \
-          > "${{ inputs.config-file }}"
-        # We have to list @commitlint/cli here otherwise `npm install` will try
-        # to remove it from the above step. There is no way to prevent this
-        # behavior that I'm aware of.
         npm install --no-save --no-package-lock --no-audit --no-fund \
```

Comment:
Might be good to call `actions/setup-node` to ensure that node is available always